Makefile

```diff
@@ -40,14 +40,7 @@ BENCHMARK_FILE_NAME ?= benchmarks.txt
 ROOT_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 BIN_DIR := $(abspath $(ROOT_DIR)/bin)
 
-BUILD_COMMIT := $(shell ./build/get-build-commit.sh)
-BUILD_TIMESTAMP := $(shell ./build/get-build-timestamp.sh)
-BUILD_HOSTNAME := $(shell ./build/get-build-hostname.sh)
```

Comment:
removed hostname, i am not sure if there is value but I can re-add if anyone thinks this is important